sail/src/main/java/org/apache/rya/rdftriplestore/inference/InferenceEngine.java

```diff
@@ -211,33 +212,9 @@ public void refreshGraph() throws InferenceEngineException {
 
             refreshOneOf();
 
-            iter = RyaDAOHelper.query(ryaDAO, null, RDF.TYPE, OWL.SYMMETRICPROPERTY, conf);
-            final Set<URI> symProp = new HashSet<>();
-            try {
-                while (iter.hasNext()) {
-                    final Statement st = iter.next();
-                    symProp.add((URI) st.getSubject()); //safe to assume it is a URI?
-                }
-            } finally {
-                if (iter != null) {
-                    iter.close();
-                }
-            }
-            symmetricPropertySet = symProp;
-
-            iter = RyaDAOHelper.query(ryaDAO, null, RDF.TYPE, OWL.TRANSITIVEPROPERTY, conf);
-            final Set<URI> transProp = new HashSet<>();
-            try {
-                while (iter.hasNext()) {
-                    final Statement st = iter.next();
-                    transProp.add((URI) st.getSubject());
-                }
-            } finally {
-                if (iter != null) {
-                    iter.close();
-                }
-            }
-            transitivePropertySet = transProp;
+            symmetricPropertySet = fetchInstances(OWL.SYMMETRICPROPERTY);
+            transitivePropertySet = fetchInstances(OWL.TRANSITIVEPROPERTY);
+            reflexivePropertySet = fetchInstances(VF.createURI(OWL.NAMESPACE, "ReflexiveProperty"));
```

Comment:
Turn the ReflexiveProperty URI into a constant so it doesn't need to be created each the engine is refreshed.